corehq/apps/registration/forms.py

```diff
@@ -487,24 +490,38 @@ class AdminInvitesUserForm(forms.Form):
     email = forms.EmailField(label="Email Address",
                              max_length=User._meta.get_field('email').max_length)
     role = forms.ChoiceField(choices=(), label="Project Role")
+    profile = forms.ChoiceField(choices=(), label="Profile", required=False)
```

Comment:
nit: just throwing this out, would the variable name `profile_id` be more clear since that's what's stored? It potentially could be more confusing, so i'm fine with leaving it as is too.

Reply:
going to leave it as is